### DISCUSSION TITLE
what different form this two case in reat-redux v7.2.3

### DISCUSSION THREAD

**Discussion by oldturkey** [reduxjs/redux#4065] [Help]
1.export default Form.create()(connect(mapStateToProps)(DynamicFieldSet)); //wrong
2.export default connect(mapStateToProps)(Form.create()(DynamicFieldSet));

this first case in react-redux V5.0.4, the props wrappedComponentRef in wrapperComponment is work great,
but after i upgrade react-redux to V7.2.3, wrappedComponentRef is undefined in first case ,it can work in second case .

can you help me ,what happend in this problem.

**Comment by timdorr** [comment]
You need to [enable the `forwardRef` option.](https://react-redux.js.org/api/connect#forwardref-boolean)
